cms/envs/common.py

```diff
@@ -465,6 +467,8 @@
 USE_I18N = True
 USE_L10N = True
 
+STATICI18N_ROOT = PROJECT_ROOT / "static"
```

Comment:
I've actually added the generation of the new files to the translation robot pull commands, so they will get committed and pushed into the repo along with the translation files. Once they're generated, they should be picked up as part of collectstatic since they'll look like normal static assets inside the repo.